unyt/mpl_interface.py

```diff
@@ -124,23 +148,16 @@ def convert(value, unit, axis):
             return converted_value
 
     class matplotlib_support:
-        """Context manager for setting up integration with Unyt in Matplotlib
-
-        Parameters
-        ----------
+        """Context manager for enabling the feature
 
-        label_style : str
-          One of the following set, ``{'()', '[]', '/'}``. These choices
-          correspond to the following unit labels:
-
-            * ``'()'`` -> ``'(unit)'``
-            * ``'[]'`` -> ``'[unit]'``
-            * ``'/'`` -> ``'q_x / unit'``
```

Comment:
Why remove the docs for `label_style`? If possible I’d like to keep them even if it duplicates the entry on the property below.

Reply:
the duplication was distracting when viewing the api documentation page - it's a tradeoff. I'll add it back.